### DISCUSSION TITLE
Unable to import geometry in fluent meshing

### DISCUSSION THREAD

**Discussion by Mr-HB833** [ansys/pyfluent#1895] [Q&A]
I have just started learning Pyfluent, so based on online cases, I am importing geometric models and want to draw grids in the fluent meshing. But I found out that I couldn't import my geometric model, but I had no problem before running the code. I was able to import the. scdoc file into fluent meshing and fluent solver. But after I finished executing the following code, I was prompted that I couldn't import the CAD geometric model, and I noticed that the startup icon of my SCDM software had changed. I used the 2022R2 version before, and I don't understand this startup icon. I don't have a clue now because I can't even import the geometric model without code. Can you give me some suggestions? What should I do?      
 ```
import ansys.fluent.core as pyfluent
meshing_session = pyfluent.launch_fluent(processor_count=10, show_gui=True, mode="meshing")
geom_filename= 'Reactor_Script_Parameters.scdoc‘
session.meshing.workflow.InitializeWorkflow(WorkflowType='Watertight Geometry')
session.meshing.workflow.TaskObject['Import Geometry'].Arguments = dict(FileName = geom_filename,LengthUnit= 'mm')
session.meshing.workflow.TaskObject['Import Geometry'].Execute()
```
![3](https://github.com/ansys/pyfluent/assets/110915109/e7f8cca0-371c-4406-aa74-f11069c09db7)

**Comment by mkundu1** [comment]
It could be an installation issue. Can you try to import the geometry in standalone Fluent meshing?

**Reply by Mr-HB833** [reply]
I'm so sorry. I cannot import any .scdoc files, so I reinstalled it. However, I am still not sure what caused it, which makes me very worried about running pyfluent statements for importing geometric models. I am afraid that this situation will happen again

**Reply by Mr-HB833** [reply]
I am worried about what happened, I now re-install, open the 2022R2 scdm software is still the old version of the boot interface, I even re-installed I can not import in the fluent meshing .scdoc file, but I try to import in the workbench,it work. I think there must be something wrong.

**Comment by raph-luc** [comment]
On Windows, Fluent unable to convert and import `.scdoc` files is not related to PyFluent, that indicates something is configured incorrectly or missing in the Ansys Fluent installation.

On Linux, I believe the `.scdoc` file format is not supported.

I just tested it on Windows with Ansys 2023 R2, and on my end PyFluent/Fluent reading `.scdoc` files works fine. The issue on your end is probably Fluent-installation related and not PyFluent. These are the commands that I used:
```
import ansys.fluent.core as pyfluent
meshing_session = pyfluent.launch_fluent(mode="meshing")
meshing_session.workflow.InitializeWorkflow(WorkflowType='Watertight Geometry')
meshing_session.workflow.TaskObject['Import Geometry'].Arguments = dict(FileName = "geometry.scdoc",LengthUnit= 'mm')
meshing_session.workflow.TaskObject['Import Geometry'].Execute()
```

The issue you described with the startup icon of your SCDM software changing, I honestly have no idea what that may be about, seems highly unlikely that is related at all to Fluent/PyFluent. Please let us know if you can reproduce and pinpoint that as an issue that is specific to Fluent/PyFluent, otherwise there is not much we can do about it.

**Reply by Mr-HB833** [reply]
Thank you very much for giving such a detailed answer to my question. I understand your meaning roughly. I 'll think about it again. Thank you very much ! ! !
